components/Asyncapi3Comparison.js

```diff
@@ -0,0 +1,549 @@
+import React, { useState } from 'react';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
Yeah, but this is an eslint error, not a logical error. So, probably, you should remvoe this line.

Reply:
Yeah, but this is an eslint error, not a logical error. So, probably, you should remvoe this line.

Reply:
Yeah, but this is an eslint error, not a logical error. So, probably, you should remvoe this line.

Reply:
So the problem is the following...

1. If I don't use it, vscode is gonna complain because you have a defined eslint configuration in the website
2. You apparently don't check with eslint on each PR, which means there are a bunch of linting errors in the website
3. I only see one location in the codebase that defines proptypes with `prop-types` import and syntax, which will remove this eslint error.

So yes, I can remove the suppression, but even better the website should probably either remove eslint completely (cause its not enforced anyway) or rewrite all components with to force `prop-types` and uphold the eslint error 😆 

So let me know which approach to take, or whether this simple suppression is good enough 😄

Reply:
We are working on this on how to use eslint configuration properly in the website repository. For now, we are right now ignoring as a part that vs code is giving eslint errors in the workspace, but these errors are not inclined towards breaking something in website rendering.

> You apparently don't check with eslint on each PR, which means there are a bunch of linting errors in the website

I completely agree on this. It has more than 100 eslint errors, but we are not concerned about it in this PR. We are on a track on how to use eslint properly in the workspace and remove the errors all together in the website repo.

Reply:
Alright, removed all the eslint ignores.